core/unit_test/TestArray.cpp

```diff
@@ -160,4 +160,23 @@ constexpr bool test_array_specialization_kokkos_swap() {
 
 static_assert(test_array_specialization_kokkos_swap());
 
+KOKKOS_FUNCTION constexpr bool test_to_array() {
```

Comment:
```suggestion
constexpr bool test_to_array() {
```

Reply:
Fixed (or will be the next time I push the repo up).